### ISSUE TITLE
y-axis inverts while editing graphs

### ISSUE THREAD

**Issue by jeisner** [mypaint/mypaint#463]
When editing graphs in the brush settings editor, the y axis sometimes inverts so that negative numbers are on top.  Perhaps this is intended as some kind of feature, but I doubt it because it's so disorienting when the whole graph flips for no apparent reason.  

As far as I can tell, this behavior is triggered when -- and only when -- you drag the *rightmost* point sufficiently downward, or drag the *leftmost* point sufficiently upward.  What does "sufficiently" mean?  I'm not sure: sometimes you have to go all the way to the bottom or the top to trigger the behavior, but sometimes going partway is enough.

**Comment by odysseywestra** [comment]
@briend I don't know what to make of this. Could you look into this when you get a chance?

**Comment by briend** [comment]
This has bugged me forever

**Comment by odysseywestra** [comment]
Yeah, I'm thinking that functionality probably should be removed just to keep the ux flow consistent. That y-axis range should be determine by the slider. Unless there a good reason for that being there.
